### ISSUE TITLE
[Suggestion] Giving Monika Nail polish!

### ISSUE THREAD

**Issue by myhoneyinthepc** [Monika-After-Story/MonikaModDev#9838]
i think it would be very cute to be able to gift Moni nail polish, in different colours. and every couple days or so it would change colours, or she could just take them off if she gets bored :P

**Comment by Wingdinggaster656** [comment]
I personally don't like nail polish that much, but this could be a good idea for people who love them.

**Comment by MikuHoney** [comment]
yeah, of course you could ask monika to take it off, and change colours, like her hair and stuff. but just to be there, maybe even get special nails for holidays, and valentines etc!

**Comment by elite198** [comment]
I'm not one to comment, but... personally I wouldn't like this (it's a funny thing that happens to me,) but it makes me uncomfortable to see people with nail polish and honestly I would feel uncomfortable if Monika used it without asking me, I would feel uncomfortable I know I'm only speaking from my point of view but I firmly believe that more people would think the same...

Also... (this is just my opinion based on what I've seen) I've seen that in the official art the other dokis sometimes wear nail polish but Monika never wears it (although a characteristic feature of Monika is that her nails are very long) which indicates to me that she doesn't usually wear nail polish (in fact Monika's opinion is one of those girls who try to look as natural as possible)

but I recommend you search on reddit, there I have seen spritepacks and even a submod that adds nail polishes

**Comment by MikuHoney** [comment]
yeah ive got nail polish on my monika, i just meant it would be cute. and if it were to be added, maybe it could start of as "[Player]... do you like nail-polish? I found some laying around and... was wondering if you would like me to put some on?" to add an option

**Comment by dreamscached** [comment]
Already something that spritepacks do
